Vagrantfile

```diff
@@ -24,6 +24,15 @@ Vagrant.configure(2) do |config|
   vm_box_version = ENV.fetch('SCF_VM_BOX_VERSION', ENV.fetch('VM_BOX_VERSION', '2.0.15'))
   vm_registry_mirror = ENV.fetch('SCF_VM_REGISTRY_MIRROR', ENV.fetch('VM_REGISTRY_MIRROR', ''))
 
+  home = "/home/vagrant"
+
+  # Set this environment variable to a directory containing shell scripts to be executed as part of
+  # the provisioning of the Vagrant machine.
+  custom_config_scripts_env = "CUSTOM_CONFIG_SCRIPTS"
```

Comment:
Does this _need_ to be an environment variable? Wouldn't a directory under the source that's `.gitignore`d be better (matching the existing thing)?

Of course, then I wonder about `personal-setup/profile.d/*`, `personal-setup/provision.d/*` instead…

Reply:
It doesn't need to be an environment variable. But it is nice to have the option of specifying a directory outside the repo tree. E.g. I can put my scripts under a private source repository that I have wherever I want on my computer.

About `personal-setup`, people that are currently using it as a file would have problems. But anyways, I don't like the gitignored folder for this purpose...